docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8615 @@
+{
```

Comment:
<p>similar comment about how to read this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>Updated </p>